src/main/java/seedu/address/ui/autocomplete/AutoCompleter.java

```diff
@@ -42,14 +42,14 @@
  */
 public class AutoCompleter {
     private static final Map<String, Set<String>> SUPPORTED_ARGUMENTS = Map.ofEntries(
-            Map.entry("addappt", Set.of("-id", "-rec", "-num", "-start")),
-            Map.entry("addshift", Set.of("-id", "-rec", "-num", "-start")),
-            Map.entry("changeappt", Set.of("-start")),
-            Map.entry("changeshift", Set.of("-start")),
-            Map.entry("edit", Set.of("-id", "-name", "-phone", "-address", "-tag", "-email")),
+            Map.entry("editappt", Set.of("-start")),
+            Map.entry("editshift", Set.of("-start")),
+            Map.entry("editdoctor", Set.of("-id", "-name", "-phone", "-address", "-email")),
+            Map.entry("editpatient", Set.of("-id", "-name", "-phone", "-address", "-tag", "-email")),
```

Comment:
can add the "-entry" to these 4 commands

Reply:
Done.